cicecore/cicedynB/infrastructure/io/io_pio2/ice_restart.F90

```diff
@@ -745,7 +745,8 @@ subroutine read_restart_field(nu,nrec,work,atype,vname,ndim3,diag, &
          status = pio_inq_varid(File,trim(vname),vardesc)
 
          if (status /= PIO_noerr) then
-            call abort_ice(subname//"ERROR: CICE restart? Missing variable: "//trim(vname))
+            call abort_ice(subname// &
+               "ERROR: CICE restart? Missing variable: "//trim(vname))
```

Comment:
why is there a question mark in this error statement?

Reply:
Not sure, I'll try to review that.